modules/ocf_tv/files/i3/config

```diff
@@ -69,14 +74,14 @@ bindsym $mod+Shift+Right move right
 bindsym $mod+semicolon split h
 
 # split in vertical orientation
-bindsym $mod+v split v
+bindsym $mod+slash split v
 
 # enter fullscreen mode for the focused container
 bindsym $mod+f fullscreen
 
 # change container layout (stacked, tabbed, toggle split)
 bindsym $mod+s layout stacking
-bindsym $mod+w layout tabbed
+bindsym $mod+t layout tabbed
```

Comment:
Again, why move away from the default here?

Reply:
I want w for web, and because mod+w for tabbed layout makes no mnemonic sense.